server.js

```diff
@@ -1,17 +1,29 @@
-const bodyParser = require("body-parser");
 const express = require("express");
 const app = express();
 const moment = require("moment");
-moment().format();
 
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(bodyParser.json());
 app.use(express.static("public"));
 
-app.post("/", function(req, res) {
-  //add request
+app.get("/:query", function(req, res) {
+  const urlString = moment.unix(req.params.query);
+  const validate = urlString.isValid();
```

Comment:
Might make sense to use a noun instead of a verb for your variable like

`valid` or `isValid` but that's just me being nit picky ;)